apps/web/src/components/views/JobListingView/JobListingFilterFunctions.tsx

```diff
@@ -0,0 +1,54 @@
+import type { JobListing } from "@dotkomonline/types"
+import type { EmploymentCheckbox } from "./JobListingView"
+export function filterJobListings(
+  jobListing: JobListing,
+  chosenLocation: string,
+  chosenEmployments: EmploymentCheckbox[],
+  searchName: string,
+  chosenSort: string
+) {
+  return (
+    filterLocation(jobListing, chosenLocation) &&
+    filterEmployment(jobListing, chosenEmployments) &&
+    filterName(jobListing, searchName)
+  )
+}
+
+export function filterLocation(jobListing: JobListing, chosenLocation: string) {
+  if (chosenLocation === "Alle") {
+    return true
+  }
+  return jobListing.locations.includes(chosenLocation)
+}
```

Comment:
It does not look like this filtering logic takes "Annet" into account? But if we use the locations that exist on the fetched job listing then we don't need "annet" anyway.